lib/modules/manager/sbt/extract.ts

```diff
@@ -336,14 +362,17 @@ function parseSbtLine(
       packageFileVersion,
     };
   }
-
   return {
     deps,
     packageFileVersion,
   };
 }
 
-export function extractPackageFile(content: string): PackageFile | null {
+export function extractPackageFile(
```

Comment:
`extractPackageFile` has special meaning and is meant to be used for all managers that don't declare `extractAllPackageFiles` for handling more complex scenarios. Therefore, it should be renamed.

```suggestion
export function extractFile(
```